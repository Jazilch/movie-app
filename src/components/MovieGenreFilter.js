'use es6';

import React from 'react';

const MovieGenreFilter = () => {
  return (
    <form method="post">
      <select name="Movie Genre Filter">
        <option value="horror">Horror</option>
        <option value="comedy">Comedy</option>
        <option value="action">Action</option>
        <option value="thriller">Thriller</option>
      </select>
    </form>
  );
};

export default MovieGenreFilter;
