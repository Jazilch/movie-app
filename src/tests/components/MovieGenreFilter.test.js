/* eslint-disable no-undef */

'use es6';

import React from 'react';
import { shallow } from 'enzyme';
import MovieGenreFilter from '../../components/MovieGenreFilter';

describe('MovieGenreFilter', () => {
  it('renders the component correctly', () => {
    const moviegenrefilter = shallow(<MovieGenreFilter />);
    expect(moviegenrefilter).toMatchSnapshot();
  });
});
