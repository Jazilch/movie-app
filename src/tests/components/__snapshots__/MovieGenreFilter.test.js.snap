// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieGenreFilter renders the component correctly 1`] = `ShallowWrapper {}`;
